intezer_sdk/_api.py

```diff
@@ -347,6 +347,12 @@ def index_by_sha256(self, sha256: str, index_as: IndexType, family_name: str = N
 
         return self._get_index_id_from_response(response)
 
+    def unset_index_by_sha256(self, sha256: str):
+        response = self.api.request_with_refresh_expired_access_token('DELETE', f'/files/{sha256}/index')
```

Comment:
Document this function, and if you can, document all of the function in this module